torch/csrc/autograd/init.cpp

```diff
@@ -742,6 +764,15 @@ static PyMethodDef methods[] = { // NOLINT
      is_inference_mode_enabled,
      METH_NOARGS,
      nullptr},
+    {"_set_multithreading_enabled",
+     set_multithreading_enabled,
+     METH_O,
+     nullptr},
+    {"_get_multithreading_enabled",
+     get_multithreading_enabled,
+     METH_NOARGS,
+     nullptr},
```

Comment:
Do this with a pybind binding on a guard class? It's a lot more compact to write.